tools/packaging/kata-deploy/Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN \
 	rm -f ${WORKDIR}/${KATA_ARTIFACTS} && \
 	apk del curl && \
 	apk --no-cache add py3-pip && \
-	pip install --no-cache-dir yq==3.2.3
+	pip install --no-cache-dir yq==3.2.3 --break-system-packages
```

Comment:
This can be considered a follow up, but do we need that specific yq, or could we bump and potentially remove the `--break-system-packages`?